lib/constants.ts

```diff
@@ -1,6 +1,7 @@
 import { v4 as uuidv4 } from "uuid";
 import { DevelopmentPlan } from "./types/timetable";
 
+// TODO: When we align the visualisation to the design, we won't need the event names here
```

Comment:
I can do that as part of [this ticket](https://trello.com/c/JMXCraid/62-user-story-adapt-planviewer-component-to-match-design) I am doing next. We wont need the event names after that change